templates/community/community.css

```diff
@@ -0,0 +1,141 @@
+.community {
+  .section.inventory {
+    padding-bottom: 15px;
+  }
+
+  .tabs-wrapper {
+    display: none;
+  }
+
+  .action-bar {
```

Comment:
The action-bar CSS can be moved to layouts.css since other templates also have it.

Reply:
This should be moved into it's own template block later on.